### DISCUSSION TITLE
Question Regarding Map / Network Routes

### DISCUSSION THREAD

**Discussion by keithdw1972** [Koenkk/zigbee2mqtt#18641] [General]
Firstly, everything is working well thanks to the developers and contributiors, great job!

I have a question, though. Whenever you press the 'Load Map' button, you get a green dialogue pop up in the top right corner that says:
Starting network scan (includeroutes: false)

It then presents a map of devices, with links.

What routes are not included? How do I set includeroutes: true? Do I want / need to? Do I care?

**Comment by jjiik** [comment]
See the same, except the only route on the map I see is the one from Coordinator to one of the smart plugs which work as routers; no other routes shown:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/66001049/d1bb58dc-e683-45d0-aecf-fc78785f66f9)

**Comment by RobSmyth** [comment]
I too would like to know what I'm missing with `includeroutes: false` or how to include routes so I can see. I notice that in the json the link routes is an empty array.

**Comment by webgress** [comment]
same here

**Comment by RobSmyth** [comment]
FYI - Searching for this again I did find a related discussion https://github.com/Koenkk/zigbee2mqtt/discussions/6260. The answer there implies it can be set in the config but does not explain how.

Another: https://github.com/nurikk/zigbee2mqtt-frontend/issues/1776

**Comment by RobSmyth** [comment]
I could not find `includeRoutes` in the docs but I did find something talking about "include routes" and a `routes` option here: https://www.zigbee2mqtt.io/guide/usage/mqtt_topics_and_messages.html#general.

When I get time I'll check what this does.

**Comment by ZhuraEN** [comment]
You just need to send JSON {"type": "raw", "routes": "true"} to the topic "zigbee2mqtt/bridge/request/networkmap"
